translations/en.json

```diff
@@ -1,7 +1,3 @@
 {
-  "Active Medications": "Active Medications",
-  "Active Programs": "Active Programs",
```

Comment:
Was this deleted by mistake? we should restore the translation.

Reply:
Ok.